src/main/java/org/folio/oaipmh/mappers/MarcXmlMapper.java

```diff
@@ -0,0 +1,9 @@
+package org.folio.oaipmh.mappers;
+
+
+/**
+ * Converts MarkJson format to MarcXML format.
+ */
+public class MarcXmlMapper extends AbstractMapper {
```

Comment:
See my comments above.  As-is, this empty class doesn't serve any practical purpose